examples/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
Do you really need `all`?
Can you use `bin` distribution?

Reply:
Back in the day -all was required for intellij completion from what I understand. Now I do see gradle compile stubs on first launch with -bin. Does that mean bin is enough for projects, but not plugin authors? If so it mostly makes sense - but I personally found the time it takes for -bin to generate stubs to be way longer (several minutes on a macbook pro) than just downloading -all so not sure it's a savings in practice. What do you think?

Reply:
Oh, I thought `-all` was generating stubs as well. I did not compare the contents, however, I thought the difference is just source code (*.java files are present in `-all`, and they are not there in `-bin`).

I think `-all` was required earlier for the IDE to navigate to Gradle sources, however, it is not needed now.

Reply:
https://twitter.com/CedricChampeau/status/1113871652809531392

If you are here, then there's Madrid JUG happening right NOW: https://www.meetup.com/madrid-gug/events/273721823

Modernizing the Groovy build by Cédric Champeau, so you can join and ask `-bin` vs `-all` :)

Reply:
I _think_ I noticed slowness with -bin with the generations of stubs that wasn't there with -all but am not sure, I'll need to try a controlled experiments to confirm.

And as much as I [love](https://www.github.com/curioswitch/curiostack/tree/master/tools%2Fgradle-plugins%2Fgradle-curiostack-plugin) Gradle I was supposed to pass out an hour ago and will promptly do so for tomorrow :) Sounds like a fun event for sure though.